changelog/objc_protocols.dd

```diff
@@ -0,0 +1,73 @@
+Add support for Objetive-C protocols
+
+It's now possible to declare Objective-C protocols. In D, this is declared using
```

Comment:
A link to what a protocol is would be nice, I have no idea what it is.

Reply:
I'll add a link. Short answer: like an interface in D.

Reply:
Fixed.